content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -436,6 +442,7 @@ model Post {
 
 model Category {
   id      String   @id @default(auto()) @map("_id") @db.ObjectId
+  name    String
```

Comment:
Although this uses the same model names, I think this is a different example where this addition would not be needed. (If we want to be consistent, then `title` is also missing in `Post`  here)

Reply:
The `name` field is used in a query further down: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#mongodb:~:text=some%3A%20%7B-,name%3A,-%7B

I was just fixing all schemas on the page to match later queries, rather than making them consistent with each other

Reply:
Ugh, ok 👍